tests/Routes/Wiki/ConversionMetricTest.php

```diff
@@ -47,28 +47,6 @@ public function testDownloadCsv()
         $response->assertSee('two.wikibase.cloud');
     }
 
-    public function testDownloadJsonWithDuplicateLifecycleEvents() {
-        $current_date = CarbonImmutable::now();
-        $wiki = $this->createTestWiki('one.wikibase.cloud', 10, 9, 2);
-        $wiki->wikiLifecycleEvents()->create([
```

Comment:
This test now has to be removed as creating a duplicate entry will not work anymore due to the added database constraint.